enterprise/models.py

```diff
@@ -708,7 +708,8 @@ def get_link_by_email(self, user_email):
             pass
 
         try:
-            return PendingEnterpriseCustomerUser.objects.get(user_email=user_email)
+            # return the first element in case of admin/learner with multiple pending enterprise associations.
+            return PendingEnterpriseCustomerUser.objects.filter(user_email=user_email).first()
```

Comment:
This is OK to return just the first element if there are multiple. This function is used downstream to determine if the user has any existing pending links to any customer.

Reply:
Then perhaps make it a boolean?

Reply:
We do expect it to the function to return either a `EnterpriseCustomerUser` or `PendingEnterpriseCustomerUser` object as we refer to its `enterprise_customer` relationship, so returning a boolean here wouldn't quite make sense.